src/main/java/md2k/mcerebrum/cstress/features/RIPFeatures.java

```diff
@@ -97,26 +99,120 @@ public RIPFeatures(DataStreams datastreams) {
 
         //Key features
 
-        double activity = datastreams.getDataPointStream(StreamConstants.ORG_MD2K_CSTRESS_DATA_ACCEL_ACTIVITY).data.get(0).value;
+        double activity = 0;//(datastreams.getDataPointStream(StreamConstants.ORG_MD2K_CSTRESS_DATA_ACCEL_ACTIVITY).data.size() == 0) ? 0 : datastreams.getDataPointStream(StreamConstants.ORG_MD2K_CSTRESS_DATA_ACCEL_ACTIVITY).data.get(0).value;
```

Comment:
Can this be moved to https://github.com/MD2Korg/stream-processor/blob/puffmarker/src/main/java/md2k/mcerebrum/cstress/features/CStressFeatureVector.java#L51